experiments/ClimaEarth/hierarchy/climate_plots.jl

```diff
@@ -0,0 +1,85 @@
+# paper_figs
```

Comment:
It might be helpful to have a more descriptive comment here explaining which paper this is for. Or even better, we can add a README inside `hierarchy/` that links to the paper (eventually :) ) and explains the goal of having these setups